src/js/over/Preview.js

```diff
@@ -48,27 +50,54 @@ class Preview extends BaseElement {
             }
         });
 
-        this.on('itemLoaded', () => this.classListRemove(this.playButton, 'pf-ui-element-hidden'));
-
         this.element.addEventListener('click', (e) => {
             const clickEvent = e || event;
             clickEvent.stopPropagation();
 
             this.classListAdd(this.element, 'pf-ui-element-hidden');
-            this.meister.play();
+            this.meister.play(true);
         });
 
         this.on('playerPlay', () => {
             this.classListAdd(this.element, 'pf-ui-element-hidden');
         });
     }
 
+    /**
+     * Makes sure that the play button is only shown when it's possible to play.
+     *
+     * @param {any} currentItem
+     * @memberof Preview
+     */
+    onPlaylistMetadata(currentItem) {
+        let isAdItem = false;
+
+        if (currentItem.parallel) {
+            // Make sure we got a item that supports vmap in the parallel items.
+            isAdItem = !!currentItem.parallel.find(parallelItem => parallelItem.type === 'vmap');
+        } else if (currentItem.type === 'aditem') {
+            // Items that simply have the aditem as type are also an ad item..
+            isAdItem = true;
+        }
+
+        // With ad items we first want to make sure we can actually load ads before showing a play button
+        if (isAdItem) {
+            // Ad blockers should not be the reason to not show the play button.
+            this.one('adBlockerDetected', () => this.classListRemove(this.playButton, 'pf-ui-element-hidden'));
```

Comment:
Potentially nice feature :) If adblocker, block playbutton for duration of ad (which cannot be displayed) ;)